R/util.R

```diff
@@ -1474,3 +1474,167 @@ one_hot <- function(df, key) {
   # Spread the column into multiple columns with name <original column name>_<original value> and value of 1 or 0.
   df %>% tidyr::spread(!!rlang::enquo(key), !!rlang::sym(tmp_value_col), fill = 0, sep = "_") %>% select(-!!rlang::sym(tmp_id_col))
 }
+
+#' Temporary work-around for https://github.com/tidyverse/dplyr/issues/977
```

Comment:
I just removed this n_distinct wrapper this morning. Maybe a merge issue? Please make sure not to revive it.

Reply:
Sorry, it seems some merge conflict issue.
I remove the n_distinct.